server/models/follow.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const FollowSchema = new Schema({
+    userId: {
+        type: String,
+        ref: 'User',
+    },
+    followers: [{
+        type: String,
+        ref: 'User',
+    }],
+    following: [{
+        type: String,
+        ref: 'User',
+    }]
```

Comment:
You're still embedding arrays of data.  My earlier comment stands and my concerns remain.  I still advise that you take a look at the CommentLikes model.  Were I do implement it, you'd have a Following model (as a noun, not a verb) with a Follower and a Followee, both User._ids, both indexed.  That's it.  I might add a date, Following._id would be generated automatically by the db and you're good to go.  Well, we still have to generate good searches in the routes, but that's the next step.